examples/roles/terraform.tfvars

```diff
@@ -83,4 +86,15 @@ roles = [
     policy_arns          = ["arn:aws:iam::aws:policy/AdministratorAccess"]
     inline_policies      = []
   },
+  {
+    name                 = "ROLE-ATTACHED-TO-AN-INSTANCE-PROFILE"
```

Comment:
I always looked at the instance profile as an attribute of the role, like with the path. With this reasoning, perhaps a better name would've been "ROLE-WITH-AN-INSTANCE-PROFILE" but I suppose folks will understand it either way :)